server/node/README.md

```diff
@@ -7,3 +7,76 @@ Start the server by running:
 ```bash
 npm start
 ```
+
+---
+
+### `GET /paypal-api/auth/browser-safe-client-token`
+
+**Description:**  
+Generates and returns a browser-safe client token for initializing the PayPal JavaScript SDK on the client side.
+
+**Responses:**
+
+- `200 OK`  
```

Comment:
Are there any error responses from these three endpoints that we should document?